cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -6,6 +6,7 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"fmt"
+	"github.com/prometheus/common/log"
```

Comment:
Are we picking up the right logger here?  Most other files just import "log"

Reply:
That is a good question for @utx0

Reply:
fixed